crates/rspack_plugin_html/src/asset.rs

```diff
@@ -62,6 +62,24 @@ impl HtmlPluginAssets {
         }
         included
       })
+      .collect::<Vec<_>>();
+
+    let sorted_entry_names = match config.chunks_sort_mode {
+      HtmlChunkSortMode::Auto => filtered_entry_names,
```

Comment:
Perhaps we can move the filtered_entry_names calculation here to improve performance when configured as "manual"

Reply:
Indeed, let me improve it.